tests/phpunit/tests/abilities-api/wpRegisterCoreAbilities.php

```diff
@@ -174,4 +174,93 @@ public function test_core_get_environment_info_executes(): void {
 		$this->assertArrayHasKey( 'wp_version', $ability_data );
 		$this->assertSame( $environment, $ability_data['environment'] );
 	}
+
+	/**
+	 * Tests that all core ability schemas only use valid JSON Schema keywords.
+	 *
+	 * This prevents regressions where invalid keywords like 'examples' are used
+	 * in schema properties (not valid in JSON Schema draft-04 used by WordPress).
+	 *
+	 * @ticket 64098
+	 */
+	public function test_core_abilities_schemas_use_only_valid_keywords(): void {
+		$allowed_keywords = rest_get_allowed_schema_keywords();
+		// Add 'required' which is valid at the property level for draft-04.
+		$allowed_keywords[] = 'required';
+
+		$abilities = wp_get_abilities();
+
+		$this->assertNotEmpty( $abilities, 'Core abilities should be registered.' );
+
+		foreach ( $abilities as $ability ) {
+			$this->assert_schema_uses_valid_keywords(
+				$ability->get_input_schema(),
+				$allowed_keywords,
+				$ability->get_name() . ' input_schema'
+			);
+			$this->assert_schema_uses_valid_keywords(
+				$ability->get_output_schema(),
+				$allowed_keywords,
+				$ability->get_name() . ' output_schema'
+			);
+		}
+	}
+
+	/**
+	 * Recursively validates that a schema only uses allowed keywords.
+	 *
+	 * @param array|null $schema           The schema to validate.
+	 * @param string[]   $allowed_keywords List of allowed schema keywords.
+	 * @param string     $context          Context for error messages.
+	 */
+	private function assert_schema_uses_valid_keywords( $schema, array $allowed_keywords, string $context ): void {
+		if ( empty( $schema ) || ! is_array( $schema ) ) {
```

Comment:
```suggestion
		if ( null === $schema ) {
```

In the case of an empty array, the `foreach()` won't do anything.